subworkflows/local/amr_annotation.nf

```diff
@@ -0,0 +1,104 @@
+// Importing the necessary modules for the workflow
+
+include { ABRITAMR_RUN } from '../../modules/nf-core/abritamr/run/main'
+include { RGI_MAIN } from '../../modules/nf-core/rgi/main/main'
+include { AMRFINDERPLUS_UPDATE } from '../../modules/nf-core/amrfinderplus/update/main'
+include { AMRFINDERPLUS_RUN } from '../../modules/nf-core/amrfinderplus/run/main'
+include { ABRICATE_RUN } from '../../modules/nf-core/abricate/run/main'
+include { ABRICATE_SUMMARY } from '../../modules/nf-core/abricate/summary/main'
+
```

Comment:
Missing `ResFinder` module, again I'm not too bothered by this. I would prioritize publishing the first release and work on adding more features/modules later on.

Reply:
and with [4504487](https://github.com/cidgoh/nf-seqqc/pull/58/commits/45044874224468086756d8388f225616d95673b6)